pkg/rpc/chunk.go

```diff
@@ -0,0 +1,23 @@
+package rpc
```

Comment:
Why this package name?  I see RPC and I think "remote procedure call" not log message forwarding

Reply:
This package contains objects to deal with incoming remote procedure calls. It’s used by the client and the server. The Chunk is part of the RPC interface, and the OutputWriter handles the output generated by the call.